analyzer/codechecker_analyzer/buildlog/log_parser.py

```diff
@@ -723,48 +723,43 @@ def __collect_transform_include_opts(flag_iterator, details):
 
     m = COMPILE_OPTIONS_MERGED.match(flag_iterator.item)
 
-    if m:
-        flag = m.group(0)
-        together = len(flag) != len(flag_iterator.item)
+    if not m:
```

Comment:
Why these files are changed? Maybe a wrong rebase? Please remove the changes from this file.